src/test/java/test/github765/ExcludeAbstractMethodsFromTemplateCallsTest.java

```diff
@@ -0,0 +1,19 @@
+package test.github765;
+
+import org.testng.Assert;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+
+
+public class ExcludeAbstractMethodsFromTemplateCallsTest extends SimpleBaseTest {
+
+    @Test
+    public void testMethod() {
+        TestNG testng = create(DuplicateCallsSample.class);
+        testng.run();
+        Assert.assertFalse(DuplicateCallsSample.messages.isEmpty(), "Method invocation log should not be empty.");
```

Comment:
The `InvokedMethodNameListener` should work too. Maybe it is not the best way to test (I'm open to any improvements) but I'd like to keep the same pattern for every tests if possible.